js/src/redux/providers/registry/middleware.js

```diff
@@ -14,13 +14,40 @@
 // You should have received a copy of the GNU General Public License
 // along with Parity.  If not, see <http://www.gnu.org/licenses/>.
 
+import { debounce } from 'lodash';
 import Contracts from '~/contracts';
 import subscribeToEvents from '~/util/subscribe-to-events';
 
 import registryABI from '~/contracts/abi/registry.json';
 
 import { setReverse, startCachingReverses } from './actions';
 
+const read = (chain) => {
+  const reverses = window.localStorage.getItem(`${chain}-registry-reverses`);
```

Comment:
1. 👍 
2.-3. Is there a specific reason for this? the `.` notation seems to be a de-facto standard. I agree with prefixing with `parity`, but why `_parity`? Does that help somehow, except worse readability?